pkg/model/bootstrapscript.go

```diff
@@ -40,7 +41,16 @@ import (
 )
 
 type NodeUpConfigBuilder interface {
-	BuildConfig(ig *kops.InstanceGroup, apiserverAdditionalIPs []string, keysets map[string]*fi.Keyset) (*nodeup.Config, *nodeup.BootConfig, error)
+	BuildConfig(ig *kops.InstanceGroup, wellKnownAddresses WellKnownAddresses, keysets map[string]*fi.Keyset) (*nodeup.Config, *nodeup.BootConfig, error)
+}
+
+// WellKnownAddresses holds known addresses for well-known services
+type WellKnownAddresses struct {
```

Comment:
Perhaps something more like a `map[wellknownservices.WellKnownService][]string`?

Reply:
Good idea - converted to a type alias on the map.